tests/infer/test_enum.py

```diff
@@ -2853,6 +2853,7 @@ def guide():
 
 @pytest.mark.parametrize("gate", [0.1, 0.25, 0.5, 0.75, 0.9])
 @pytest.mark.parametrize("rate", [0.1, 1., 3.])
+@pytest.mark.skip(reason="FIXME: regression in pytorch-1.2")
```

Comment:
Let's try to fix this in 0.4, since Paul wanted to use this for rate modeling.

Reply:
Yes, still looking into this and some jit failures.